src/client.py

```diff
@@ -70,8 +70,9 @@ def __init__(self, read_env_data=True, verbose=True):
 
         self._hive_engine = self.flagged.get_engine()
         engine_url = self._hive_engine.url
-        self.flags = Flags(verbose=verbose, engine=engine_url)
-        self.service_periods = Service_Periods(verbose=verbose, engine=engine_url)
+        self.flags = Flags(pipe_user, pipe_passwd, pipe_hostname, pipe_db_name, pipe_schema, verbose=verbose)
+        #self.service_periods = Service_Periods(verbose=verbose, engine=engine_url)
+        self.service_periods = Service_Periods(pipe_user, pipe_passwd, pipe_hostname, pipe_db_name, pipe_schema, verbose=verbose)
```

Comment:
Can you explain this change to me?

Reply:
Hive wasn't created in my db (IlliaS_db) without specifying all the details (like for ctran)

Reply:
What was the error you were getting?

Reply:
I don't remember tbh, it was just not creating hive for me if I recall correcly

Reply:
As it is now, the user will need to put the same information in several times to connect to the same database if config can't read the data. Could you give me a specific error you were hitting so we can figure it out, or can you adjusting so they don't have to do this?

Reply:
I'll have to look at it tomorrow to figure out. I think if in testing you remove the last 2 tests (which remove the schemes), and put back verbose stuff (so no pipe_user etc), you will hit the same errors. Just change config to your acc (or just look into my db)